roles/libvirt_setup/tasks/main.yml

```diff
@@ -36,6 +36,9 @@
     name: provision
     xml: '{{ lookup("template", "../templates/network.xml.j2") }}'
 
+- name: 'make provisioning network autostart on reboot'
+  file: state=link src=/etc/libvirt/qemu/networks/provision.xml path=/etc/libvirt/qemu/networks/autostart/provision.xml
```

Comment:
Why not `virsh net-autostart provision`?

Reply:
You could also add ```autostart: yes``` to the above ```virt_net``` definition for the same effect.

Reply:
sadly either didn't work, I think it's a bug in libvirt, I see the network is set to autoprovision but the symlink is missing and it does not start after HW is rebooted, I'll add extra comment

Reply:
also from what I read, `autostart: yes` is default